diff -urN dreamchess-0.2.0/configure dreamchess-0.2.0-patched/configure
--- dreamchess-0.2.0/configure	2007-11-16 22:42:29.000000000 +1000
+++ dreamchess-0.2.0-patched/configure	2012-03-14 17:11:22.138483901 +1100
@@ -6846,7 +6846,7 @@
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lmxml  $LIBS"
+LIBS="-lmxml -pthread -lm $LIBS"
 cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
@@ -6906,7 +6906,7 @@
 #define HAVE_LIBMXML 1
 _ACEOF
 
-  LIBS="-lmxml $LIBS"
+  LIBS="-lmxml -pthread -lm $LIBS"
 
 else
   { { echo "$as_me:$LINENO: error: Cannot find Mini-XML library." >&5
